app/src/main/res/values-hi/strings.xml

```diff
@@ -168,15 +168,15 @@
     <string name="view_repayment">पुनर्भुगतान शेड्यूल देखें</string>
     <string name="share_qr_code">क्यूआर कोड साझा करें</string>
     <string name="monitor">निगरानी करें</string>
-    <string name="view_qr_code">इस खाते के लिए क्यूआर कोड देखें</string>
+    <string name="view_qr_code">इस खाते के लिए क्यू.आर कोड देखें</string>
     <string name="last_trans">पिछले लेनदेन का विवरण</string>
     <string name="passcode_setup">लॉगिन करने के लिए एक पासकोड सेटअप करें</string>
     <string name="error_fetching_third_party_transfer_template">थर्ड पार्टी ट्रांसफर टेम्प्लेट को लाने में त्रुटि</string>
     <string name="choose">एप्लिकेशन चुनें</string>
 
     <string name="fetching_client">क्लाइंट ला रहा है</string>
 
-    <string name="not_contain_username">Spaces</string>
+    <string name="not_contain_username">रिक्त स्थान</string>
```

Comment:
Please check in which context this string is used in the application, I feel like It is just to follow non-hardcoded string pattern.

Reply:
I resolved it now.